backend/src/main/java/de/neuefische/teddyscoronadiaries/service/CovidService.java

```diff
@@ -0,0 +1,64 @@
+package de.neuefische.teddyscoronadiaries.service;
+
+import de.neuefische.teddyscoronadiaries.covid19api.model.ConfirmedCase;
+import de.neuefische.teddyscoronadiaries.covid19api.service.Covid19ApiService;
+import de.neuefische.teddyscoronadiaries.model.covid.IncidenceDetails;
+import de.neuefische.teddyscoronadiaries.model.covid.IncidenceLevel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class CovidService {
+
+    private final Covid19ApiService covid19ApiService;
+    private final Instant quarantineStart = Instant.ofEpochSecond(1584230400L);
+    private final float inhabitantsGermany = 83100000;
```

Comment:
```suggestion
    private final int inhabitantsGermany = 83100000;
```
or do we split some inhabitants 😉

Reply:
maybe you could also move it to a config or you could define it as const 
```suggestion
    private static final int INHABITANTS_GERMANY = 83100000;
```

Reply:
🤣  dogs and children under 3 count as half people